webui/src/Controls/InternalModuleField.tsx

```diff
@@ -225,6 +227,28 @@ export const InternalPageIdDropdown = observer(function InternalPageDropdown({
 	}
 })
 
+// Formatting for variable pulldown options:
+const CustomOption = React.memo((props: OptionProps<DropdownChoiceInt>) => {
+	const { data } = props
+	return (
+		<SelectComponents.Option {...props} className={(props.className ?? '') + 'variable-dropdown-option'}>
+			<span className="var-name">{data.value}</span>
+			<span className="var-label">{data.label}</span>
+		</SelectComponents.Option>
+	)
+})
+
+// Formatting for variable "single value" (shown when dropdown is closed) -- uses a different CSS class
+const CustomSingleValue = React.memo((props: OptionProps<DropdownChoiceInt>) => {
```

Comment:
As with elsewhere, this feels rather squeezed. Visually it doesnt match how we do padding elsewhere.

I know there have been some comments that we have too much spacing/padding, but I think being consistent is more important